public/js/main.js

```diff
@@ -36,19 +37,27 @@ const actions = bindActionCreators(require("./actions"), store.dispatch);
 // global for debugging purposes only!
 window.store = store;
 
-connectClient(response => {
-  actions.newTabs(response.tabs);
+connectClient(tabs => {
+  actions.newTabs(tabs);
 
   // if there's a pre-selected tab, connect to it and load the sources.
   // otherwise, just show the toolbox.
   if (hasSelectedTab()) {
     const selectedTab = getSelectedTab(store.getState().tabs.get("tabs"));
-    debugTab(selectedTab.toJS(), actions).then(renderToolbox);
+    const tab = selectedTab.get("firefox") || selectedTab.get("chrome");
```

Comment:
I'll fix this when i tackle debugging a tab for chrome. Want to wait because there might be more there.